lib/dependency-manager-adapters/npm.js

```diff
@@ -16,11 +16,14 @@ module.exports = CoreObject.extend({
   },
   useYarnCommand: false,
   yarnLock: 'yarn.lock',
+  yarnLockBackupFileName: 'yarn.lock.ember-try',
   configKey: 'npm',
   packageJSON: 'package.json',
   packageJSONBackupFileName: 'package.json.ember-try',
   nodeModules: 'node_modules',
   nodeModulesBackupLocation: '.node_modules.ember-try',
+  NPMShrinkWrap: 'npm-shrinkwrap.json',
+  NPMShrinkWrapBackupFileName: 'npm-shrinkwrap.json.ember-try',
```

Comment:
What about package-lock.json?

Reply:
Ya, good catch! We should definitely retrieve and preserve `package-lock.json` in the same way that we do `yarn.lock`.